src/slidepane/SlidePane.ts

```diff
@@ -156,7 +189,10 @@ export default class SlidePane extends SlidePaneBase<SlidePaneProperties> {
 
 		const contentClasses = [
 			css.pane,
-			align === Align.left ? css.left : css.right,
+			align === Align.left ? css.left : null,
```

Comment:
This is where I originally tried to get clever using string enum values above for `Align`. I couldn't get it to work because of typings and indexes. Happy to try a less verbose approach though.